js/main.js

```diff
@@ -112,7 +127,19 @@ function viewSwapping(target) {
   } else if (data.view === 'view-profile') {
     renderProfile();
     $views.editProfile.className = 'edit-profile hidden';
+    $views.entryCreate.className = 'entry-create hidden';
+    $views.entryNew.className = 'entries hidden';
     $views.viewProfile.className = 'view-profile';
+  } else if (data.view === 'entries') {
+    $views.editProfile.className = 'edit-profile hidden';
+    $views.viewProfile.className = 'view-profile hidden';
+    $views.entryCreate.className = 'entry-create hidden';
+    $views.entryNew.className = 'entries';
+  } else if (data.view === 'create-entry') {
+    $views.editProfile.className = 'edit-profile hidden';
+    $views.viewProfile.className = 'view-profile hidden';
+    $views.entryNew.className = 'entries hidden';
+    $views.entryCreate.className = 'entry-create';
```

Comment:
You can merge this pr and move on, but before you do I want you to attempt to do this in a loop.
The easiest thing would be to hide all of the views, and then unhide the one related to the view swap
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList